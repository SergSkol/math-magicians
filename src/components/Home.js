import React from 'react';

const Home = () => (
  <div className="homeContent">
    <h2>Welcome to our page!</h2>
    <p className='homeText'>
      This web site was created follow the tutorial:
      React Tutorial From Scratch: A Step-by-Step Guide (2021)!
      https://ibaslogic.com/react-tutorial-for-beginners/
      This tutorial, divided into several parts, is very comprehensive,
      and covers all main concepts about React, from the very basics to some intermediate topics.
    </p>
  </div>
);

export default Home;
