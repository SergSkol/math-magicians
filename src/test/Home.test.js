import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('Test home page with snapshot', () => {
  const tree = renderer
    .create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
