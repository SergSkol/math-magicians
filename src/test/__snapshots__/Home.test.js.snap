// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test home page with snapshot 1`] = `
<div
  className="homeContent"
>
  <h2>
    Welcome to our page!
  </h2>
  <p
    className="homeText"
  >
    This web site was created follow the tutorial: React Tutorial From Scratch: A Step-by-Step Guide (2021)! https://ibaslogic.com/react-tutorial-for-beginners/ This tutorial, divided into several parts, is very comprehensive, and covers all main concepts about React, from the very basics to some intermediate topics.
  </p>
</div>
`;
